app/controllers/users_controller.rb

```diff
@@ -3,7 +3,7 @@ class UsersController < ApplicationController
 
   before_filter :check_user_status, :only => [:edit, :update]
   before_filter :load_user, :except => [:activate, :create, :delete_confirmation, :index, :new]
-  before_filter :check_ownership, :except => [:activate, :browse, :create, :delete_confirmation, :index, :new, :show]  
+  before_filter :check_ownership, :except => [:activate, :browse, :create, :delete_confirmation, :index, :new, :show]
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer `before_action` over `before_filter`.